resqpy/olio/relperm.py

```diff
@@ -0,0 +1,316 @@
+"""relperm.py: class for storing and retrieving dataframes of relative permeability
+data as RESQML objects.
+
+   note that this module uses the obj_Grid2dRepresentation class in a way that was not envisaged
+   when the RESQML standard was defined; software that does not use resqpy is unlikely to be
+   able to do much with data stored in this way
+"""
+import numpy as np
+import pandas as pd
+from scipy import interpolate
```

Comment:
To avoid adding extra dependencies to resqpy, could we use `numpy.interp` rather than `scipy.interpolate`?

Reply:
Done!